meson.build

```diff
@@ -885,15 +885,6 @@ config_extra_h = configuration_data()
 
 config_extra_h.set_quoted('BINDIR',            nm_bindir)
 config_extra_h.set_quoted('DATADIR',           nm_datadir)
-if enable_dhclient
```

Comment:
is this patch right? config-extra.h.meson still contains `mesondefine DHCPCANON_PATH`. Also, don't you need to set_quoted() now somewhere else? Is this fixed by the following commit? If so, it seems this patch is broken and should be merged with the following commit.

Reply:
I removed the leftover defines from config-extra.h.meson now.

The `config_h.set_quoted('DHCPCANON_PATH', dhcpcanon.path())` is already there (previously the path was added to both config.h and config-extra.h)